### PR TITLE
Add dependency operators for deploying RHOAI addon

### PR DESCRIPTION
**Pull request by aloganat**

### CODE REVIEW

**Review by tarukumar — APPROVED** [review]

File: ods_ci/utils/scripts/ocm/ocm.py [hunk 1]
```diff
@@ -1198,7 +1207,37 @@ def install_openshift_isv(
...
+                yaml.dump(newdct, f)
+
+        if operator_name == "serverless":
+
+            replace_vars = {
+                "ISV_NAME": "serverless-operators",
+                "NAMESPACE": "openshift-serverless"
+            }
+            template_file = "resource.jinja"
+            file_path1 = "resource.yaml"
```
Comment (jstourac) [inline ods_ci/utils/scripts/ocm/ocm.py:1225]:
```suggestion
            resource_file = "resource.yaml"
```

File: ods_ci/utils/scripts/ocm/ocm.py [hunk 2]
```diff
@@ -1198,7 +1207,37 @@ def install_openshift_isv(
...
+            self._render_template(template_file, file_path1, replace_vars)
+
+            def yaml_loader(filepath):
+                with open(filepath,'rb')as file_descriptor:
```
Comment (jstourac) [inline ods_ci/utils/scripts/ocm/ocm.py:1229]:
```suggestion
                with open(filepath,'rb') as file_descriptor:
```

File: ods_ci/utils/scripts/ocm/ocm.py [hunk 3]
```diff
@@ -1198,7 +1207,37 @@ def install_openshift_isv(
...
+                    data = yaml.load(file_descriptor, Loader=yaml.SafeLoader)
+                return data
+
+            data1 = yaml_loader(file_path1)
```
Comment (jstourac) [inline ods_ci/utils/scripts/ocm/ocm.py:1233]:
```suggestion
            serverless_data = yaml_loader(file_path1)
```

File: ods_ci/utils/scripts/ocm/ocm.py [hunk 4]
```diff
@@ -1198,7 +1207,37 @@ def install_openshift_isv(
...
+            data2 = yaml_loader(output_file)
```
Comment (jstourac) [inline ods_ci/utils/scripts/ocm/ocm.py:1234]:
```suggestion
            output_data = yaml_loader(output_file)
```

**Review by jstourac — APPROVED** [review]
Well, if it's possible, I would change the naming of some properties so they are more understandable. But if this is something urgent, I don't insist.

**Pull request merged**